client/state/products-list/selectors.js

```diff
@@ -27,14 +27,26 @@ export function getAvailableProductsList( state ) {
 }
 
 /**
- * Returns the display price of a product
+ * Retrieves the product with the specified slug.
  *
- * @param {Object} state The Redux state tree
- * @param {string} productSlug The internal product slug, eg 'jetpack_premium'
- * @return {string} The display price formatted in the user's currency, eg "A$29.00"
+ * @param {object} state - global state tree
+ * @param {string} productSlug - internal product slug, eg 'jetpack_premium'
+ * @returns {?object} the corresponding slug, or null if not found
```

Comment:
Shouldn't this be _the corresponding **product**, or null if not found_ ?

Reply:
Of course, good catch 👍.